codecov.yml

```diff
@@ -0,0 +1,26 @@
+# Sample CodeCov config for private repo
+codecov:
+  bot: luongvo
```

Comment:
possibly not need this

Reply:
I put a comment above ☝️ , we need it for the private repository. We create this template for any project, right? Should we go to resolve issues later with private repo integration without this config? 🤓

Reply:
@luongvo I'm not sure if it should be the case for every project 🤔 . Check #21 I think I mentioned clearly that it should be scoped for this template_project only - to make sure that our code meets the standard before being used by someone else. Now that we are having:
- Flagging system for add-on features
- Incoming `example/` codebase

I think we need to plan strategically what will happen next. I.e: if the client doesn't want to use this service, we will have to remove this config. Then should it be flag as an optional feature? etc...

Reply:
@luongvo I think we should let this config as an optional feature. This file could be added to the project if the flag is true imo 🙆‍♂️

Reply:
Okay, let me remove the CodeCov integration out of this PR and put the CodeCov integration ticket back to In Dev

Reply:
@sleepylee @manh-t updated